app/models/tasks/copy_image_task.rb

```diff
@@ -1,17 +1,24 @@
 class Tasks::CopyImageTask < ::Task
   before_save do
     if image_resource && status_changed?
-      image_resource.update!(status: status)
       if complete?
-        image_resource.update!(url: image_resource.published_url)
-        podcast.try(:publish!)
+        meta = porter_callback_image_meta
+        if meta
+          update_image!(meta.merge(url: image_resource.published_url))
```

Comment:
Copied from the Content (audio) task with 1 exception...

There's a chance that the image fails validation, because its height/width are too small.  So for now, just catching that and setting status to "error".  Will figure out a better way to relay to the user later.